services/SeomaticService.php

```diff
@@ -2478,11 +2478,7 @@ public function getMainEntityOfPageJSONLD($meta, $identity, $locale, $isMainEnti
      */
     protected function getAggregrateRatingForJsonLD(ReviewsServiceInterface $plugin, array $mainEntityOfPageJSONLD, BaseElementModel $element, BaseModel $settings)
     {
-        if (isset($element->productCategory) &&
-            $element->productCategory->value === 'homeCharge' &&
-            isset($element->productDisplayRating) &&
-            isset($element->productDisplayRating->value) &&
-            $element->productDisplayRating->value)
+        if (isset($element->productCategory) && $element->productCategory->value === 'homeCharge')
```

Comment:
`homeCharge` is a bit specific to us, is there a way to do this where it comes from some settings or config?

Reply:
I'm gonna check that 👍